[FM v0.3.0]-- Support specifying objects as unmanaged objects (#135)
streamnative/function-mesh-website: docs/connectors/io-crd-config/sink-crd-config.md

| `cleanupSubscription` | Configure whether to clean up subscriptions. |
| `subscriptionPosition` | The subscription position. |

## Annotations

In Kubernetes, an annotation defines an unstructured Key Value Map (KVM) that can be set by external tools to store and retrieve metadata. `annotations` must be a map of string keys and string values. Annotation values must pass Kubernetes annotations validation. For details, see [Kubernetes documentation on Annotations](https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/#syntax-and-character-set).

> This example shows how to use an annotation to make an object unmanaged. Therefore, the Controller can skip unmanaged objects when checking for the annotation in the reconciliation loop.

Reviewer: ```suggestion
This example shows how to use an annotation to make an object unmanaged. Therefore, the Controller will skip reconciling unmanaged objects in reconciliation loop.
```

same for other places

Author: @MarvinCai Thanks for your comments. Docs are updated accordingly. Could you please take another look and feel free to approve it if there are no more comments. Thanks
